webapp/components/test-runs-query.html

```diff
@@ -205,14 +206,25 @@
           this.setProperties(batchUpdate);
         }
 
-        computeResultsRangeMessage(from, maxCount) {
-          if (from) {
-            var diff = Math.floor((Date.now() - from) / 86400000);
-            return `Showing test runs from the last ${diff} days`;
+        computeResultsRangeMessage(sha, productSpecs, from, to, maxCount) {
+          if (sha) {
+            return `Showing results from revision ${sha}`;
+          } else if (from) {
+            const max = to || Date.now();
+            var diff = Math.floor((max - from) / 86400000);
+            const shortDate = date => date.toLocaleDateString(navigator.language, {
+              month: 'short',
+              day: 'numeric',
+            });
+            return to
+              ? `Showing test runs from ${shortDate(from)} to ${shortDate(to)}`
```

Comment:
If the locale isn't English this will be a mix of languages. `navigator.language` is used in a lot of contexts like this now, but maybe they should all be the "en-US" or something that fits together with the rest of the UI? (Seems unlikely we'll ever localize the whole thing.)

Reply:
Done.